pkg/version/release.go

```diff
@@ -6,7 +6,7 @@ package version
 var Version = "0.37.0"
 
 // PreReleaseID can be empty for releases, "rc.X" for release candidates and "dev" for snapshots
-var PreReleaseID = "dev"
+var PreReleaseID = ""
```

Comment:
this shouldn't change

Reply:
good spot! I think this was caused by me not doing these manual steps https://github.com/weaveworks/eksctl/blob/35400fb2cda8dffb97243fc555e8c44d12c71724/.github/workflows/release-merge.yaml#L24-L25

Reply:
fixed